contracts/Tokens/Prime/PrimeLiquidityProvider.sol

```diff
@@ -325,7 +332,7 @@ contract PrimeLiquidityProvider is
 
         _ensureTokenInitialized(token_);
 
-        uint256 blockNumber = getBlockNumber();
+        uint256 blockNumber = getBlockNumberOrTimestamp();
```

Comment:
```suggestion
        uint256 blockNumberOrTimestamp = getBlockNumberOrTimestamp();
```
I think we can change the name of local variables to avoid any confusion.

Reply:
I would update comments and internal variable names. We can review specific cases if needed